DESCRIPTION

```diff
@@ -11,17 +11,21 @@ License: Apache License (== 2.0) | file LICENSE
 Depends:
     R (>= 3.2.0)
 Imports:
-    dplyr (>= 0.4.3),
+    dplyr (>= 0.7.1),
+    ggplot2 (>= 2.0.0),
+    ggmap (>= 2.6.1),
+    httr (>= 1.3.1),
+    lubridate (>= 1.5.0),
     rgdal (>= 1.1-3),
```

Comment:
I'm thinking we should move rgdal and sp to Suggests as they're only used for the one function (`utm_dd`). Then at the top of the body of `utm_dd` use:
```
if (!requireNamespace("rgdal") || !requireNamespace("sp")) {
  stop("You need the sp and rgdal packages installed to use this function")
}
```

Thoughts?

Reply:
Definitely, especially as rgdal and sp aren't lightweight packages